src/styles/actor/character/_proficiency-ranks.scss

```diff
@@ -44,9 +48,6 @@
 
 select.pf-rank {
     cursor: pointer;
-    line-height: 0.9em;
-    padding: 0;
-    text-align: center;
```

Comment:
also test on firefox if you can. A lot of the reason the pf-rank is the way it is, is because select rendering is pretty screwy on different browsers (they all handle it differently). There's also tiny minor font alignment and rendering differences.